README.md

```diff
@@ -76,6 +76,35 @@ chifra blocks 12345
 6. To make deep data queries, [get the index](https://trueblocks.io/docs/install/get-the-index/)
 7. To explore the data visually, [install the explorer application](https://trueblocks.io/docs/install/install-explorer/).
 
+## Docker Container
+
+A `Dockerfile` is included in this repo as an example for creating a Docker image.
+
+1. Build a docker image (example tagged with `latest`)
+  ```bash
+  docker build . --tag=trueblocks-core:latest
```

Comment:
Can I ask you about this line? Does it always take the latest 'master' branch? Or maybe the most recently tagged commit? (Sorry not real good at `git` either.)

Reply:
The assumption here is that they will clone this repository and run the Dockerfile from whatever version the have checked out. The Dockerfile does not clone, it adds the files with `ADD . /trueblocks-core`